### DISCUSSION TITLE
Customization in Title Bar

### DISCUSSION THREAD

**Discussion by NaveenBadal** [dotMorten/WinUIEx#103] [Q&A]
Hello, 

I am new to WinUI and strugging to do some basic things in that. Any help is appreciated. 

1.) Is there any way to show or hide default icon from the title bar in WinUI 3. I tried to set the null in this.SetIcon(null). But the default icon is still showing in the Title Bar. 

2.) Is the SVG Icon not supported in this.SetIcon(iconPath) or in the TaskBarIcon xaml property. I got exception when I am trying to set the svg icon. 

3.) When using the ExtendsContentsIntoTitleBar = true, Minimize and Maximize button is still showing in the titlebar, even if I set the : 
this.IsMinimizable = false; or
this.SetIsMinimizable(false);  //Same with the maximizable.

**Comment by dotMorten** [comment]
I'll have to get back to you on 1.
2) No, it has to be a .ico file
3) ExtendsContentsIntoTitleBar just allows you to put xaml all the way there. It'll still go up under the window buttons. The methods you refer to disables the buttons, it doesn't remove them. You'll have to use a different window presenter to get rid of those. This is more of a WinUI question though and not releated to WinUIEx.

**Reply by NaveenBadal** [reply]
Hi ,

Thank you for the response.
1.) Regarding Point 1, Sure will wait for the response.

2.) Regarding Point 3, Can you please tell me how to achieve this. As I am totally new to WinUI, I am not aware of this manager and how to do that. 

Please give me a little snippet or sample code, how we can achieve this.

Thank you.

**Reply by dotMorten** [reply]
What I meant is you want to post that in the WinUI repos or Microsoft/StackOverflow forums. I’d be happy to help discuss issues around WinUIEx here but don’t want it to turn into a general WinUI support channel.

**Reply by NaveenBadal** [reply]
Okay, Thank you for the suggestion. I have put the question on the StackOverFlow . Here is the link of this : https://stackoverflow.com/questions/75205781/winui-3-title-bar-customization
